messenger.rst

```diff
@@ -344,59 +285,69 @@ instead of a class name:
         $container->loadFromExtension('framework', [
             'messenger' => [
                 'routing' => [
-                    'My\Message\Message' => 'another_transport',
-                    '*' => 'your_transport',
+                    'App\Message\AbstractAsyncMessage' => 'async',
+                    'App\Message\AsyncMessageInterface' => 'async',
+                    'My\Message\ToBeSentToTwoSenders' => ['async', 'audit'],
                 ],
             ],
         ]);
 
-A class of messages can also be routed to multiple senders by specifying a list:
+Doctrine Entities in Messages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. configuration-block::
+If you need to pass a Doctrine entity in a message, it's better to pass the entity's
```

Comment:
I'm probably wrong but ... isn't Messenger used in distributed systems? The Doctrine ID may only exist in the machine that sends the message, not in the one which processes it, right?

Reply:
Hmm, I think this recommendation depends very much on the use-case. Most often in distributed systems, you don't really care about the object, just an identifier is enough. In this example (sending an email), it's probably more important to pass the email of the user (and maybe it's name), besides only the ID.

What do you think about modifying the recommendation to simply "Do not pass the entity directly into the message, try to only pass relevant information"?

Reply:
I added a note about passing relevant info - I think it's a good point